.scalafmt.conf

```diff
@@ -0,0 +1,13 @@
+style = defaultWithAlign
+
+align                      = some
+align.tokens               = [off]
```

Comment:
`align.tokens = [{code = "=>", owner = "Case"}]` [might be nice](https://scalameta.org/scalafmt/docs/configuration.html#aligntokens)

Reply:
thanks, I was looking for that and thought it would be covered by `align = some`.
added in b042e06